orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/WebhookResponseProcessor.java

```diff
@@ -181,7 +183,7 @@ private TaskResult processResponse(ResponseEntity response) {
     Map<String, Object> stageOutput = new HashMap<>();
     var webhookOutput = new WebhookStage.WebhookResponseStageData();
     stageOutput.put("webhook", webhookOutput);
-    webhookOutput.setStatusCode(response.getStatusCode());
+    webhookOutput.setStatusCode(HttpStatus.valueOf(response.getStatusCode().value()));
```

Comment:
```suggestion
    webhookOutput.setStatusCode(response.getStatusCode());
```
works for me locally.